wrappers/mason

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# if there is a CHPL_HOME, use that to find mason
+# if that fails, fall back to the mason in path
+
+mason="none"
+
+if [ -n "$CHPL_HOME" ]; then
+  chpl_python=$($CHPL_HOME/util/config/find-python.sh 2> /dev/null)
+  exit_code=$?
+  if [ $exit_code -eq 0 ]; then
+    chpl_bin_subdir=$($chpl_python $CHPL_HOME/util/chplenv/chpl_bin_subdir.py)
```

Comment:
Check to make sure if `chpl_bin_subdir` worked with another `  if [ $exit_code -eq 0 ]; then` perhaps?